Update who_with_quizlink.md (#98)
esciencecenter-digital-skills/workshop-metadata: ds-dl-intro/who_with_quizlink.md

This course is for you if:

You have been using machine learning but are starting to dive in to the field of deep learning and you want to learn:

> - What is deep learning?
> - How to structure a deep learning project?
> - What are the different techniques used in deep learning?
> - How to address common pitfalls in a deep learning workflow?

Reviewer: - Prepare input data for use for deep learning
> - Design and train a Deep Neural Network
> - Troubleshoot the learning process
> - Measure the performance of the network
> - Visualizing data and results
> - Re-use existing network architectures with and without pre-trained weights

```suggestion
- What is deep learning?
- How to structure a deep learning project?
- How to prepare input data for use for deep learning?
- How to design and train a basic neural network using Keras?
- How to evaluate a neural network?
- How to address common pitfalls in a deep learning workflow?
```
I think this makes it a bit more specific

This course assumes that you are currently comfortable with:
- Basic Python programming skills and familiarity with the Pandas package.

> - Basic knowledge on Machine learning, including the following concepts: Data cleaning, type of problems (regression, classification), metrics (accuracy, root-mean-square-error, etc.).

Reviewer: ```suggestion
- Basic knowledge on machine learning, including the following concepts: data cleaning, type of problems (regression, classification), metrics (accuracy, root-mean-square-error, etc.).
```

> * please note that some relevant concepts will also be covered during the course.

Reviewer: ```suggestion
* please note that some of the traditional machine learning concepts will be covered during the course, since they are also relevant for deep learning.
```

This course is not for you if:

- You are well-versed with the topics covered in this course (We strongly recommend that you go through the syllabus to identify your suitability to the course)
- You have little to no experience with machine learning and/or python programming:
Please take this [one-minute quiz](https://forms.office.com/Pages/ResponsePage.aspx?id=zOo6qgdjskKsBXh91cMldBo0Sr0r9LlAky4oLGSYS6RUMElCSVkzVjBSVlQ2SUJJWFFUU1o1OFVUVy4u),
if you answered "No" to most of the questions, this workshop may be too advanced for you, 
and you may want to consider unregistering from this workshop and signing up in the future, when you have the prerequired knowledge. 
You could consider following a [data carpentry workshop](https://datacarpentry.org/python-ecology-lesson/) to learn about data preprocessing 
and/or learn more about machine learning in this course. Check  our upcoming workshops [here](https://www.esciencecenter.nl/events/?f=workshops). 
Please contact training@esciencecenter.nl if you have any questions about the prerequisites or course content.

This course does not cover the following topics (in detail):

> - transfer learning
> - hyper-parameter search
> - Advanced deep learning topics such as few-shot learning, large-language model, etc.
> * For a complete list of the topics covered in the course, please refer to the syllabus.

Reviewer: ```suggestion
- transfer learning
- hyper-parameter search
- advanced deep learning topics such as few-shot learning, large-language model, etc.
* For a complete list of the topics covered in the course, please refer to the syllabus.
```
